### DISCUSSION TITLE
Replace symbols in String

### DISCUSSION THREAD

**Discussion by dvgmdvgm** [vfsfitvnm/frida-il2cpp-bridge#118] [Q&A]
Hello!
I need to replace some part of string but cannot with default `replace` like:
`path = path.replace("sorted", "non-sorted");`

have this error when rebuildig script 
`Property 'replace' does not exist on type 'String'`
`Property 'Replace' does not exist on type 'String'`

**Comment by vfsfitvnm** [comment]
A `Il2Cpp.String` is not a JavaScript string.
```ts
path.content = path.content.replace("..", "...");
```

However, I suggest you to use an IDE (e.g. vscode), so you can easily see which method you can call given an object

**Comment by dvgmdvgm** [comment]
@vfsfitvnm Thank you fro quick response..
But get this error
`Object is possibly 'null'.`
By following code

```
Assets.methods[".ctor"].implementation = function (path: Il2Cpp.String, options: Il2Cpp.Object): void {
		path.content = path.content.replace("sorted", "non-sorted");
};
```

**Comment by vfsfitvnm** [comment]
Hey, I apologize but I have to close this. It's a js/ts/general programming related "error" - search it on google and you will know how to solve it...
